src/components/Table/GridTable.test.tsx

```diff
@@ -913,9 +922,9 @@ describe("GridTable", () => {
     // When we filter by 'foo'
     const r = await render(<GridTable filter="foo" columns={nestedColumns} rows={rows} />);
     // Then we show the header
-    expect(cell(r, 0, 1)).toHaveTextContent("Name");
+    expect(cell(r, 0, 2)).toHaveTextContent("Name");
```

Comment:
With a new column for testing "select", some existing assertions moved their column over by 1.